modules/3d-tiles/src/lib/utils/morton.ts

```diff
@@ -0,0 +1,78 @@
+// Morton Index specification - https://en.wikipedia.org/wiki/Z-order_curve
+// Code is taken from - https://github.com/thi-ng/umbrella/tree/develop/packages/morton
```

Comment:
Specify under which OSS license this file was "forked" (Apache 2.0 in this case, which is fine).

Reply:
Adding a util is fine, but please make an effort to also add a test: https://github.com/thi-ng/umbrella/tree/develop/packages/morton/test

Reply:
Removed this part